packages/next/src/generators/application/lib/update-jest-config.ts

```diff
@@ -13,6 +13,9 @@ export function updateJestConfig(host: Tree, options: NormalizedSchema) {
       'transform: {',
       "transform: {\n    '^(?!.*\\\\.(js|jsx|ts|tsx|css|json)$)': '@nrwl/react/plugins/jest',"
     )
-    .replace(`'babel-jest'`, `['babel-jest', { presets: ['next/babel'] }]`);
+    .replace(
+      `'babel-jest'`,
+      `['babel-jest', { presets: ['@nrwl/next/babel'] }]`
```

Comment:
@jaysoo Should `plugins` and preset options to be migrated too?

For example, when we have following `.babelrc`,
```js
{
  "presets": [
    [
      "@nrwl/next/babel",
      {
        "preset-react": {
          "runtime": "automatic",
          "importSource": "@emotion/react"
        }
      }
    ]
  ],
  "plugins": ["@emotion/babel-plugin"]
}
```

the `transform` option should be like this? (or just keep it `'^.+\\.[tj]sx?$': 'babel-jest'` ?)


```js
transform: {
  '^(?!.*\\.(js|jsx|ts|tsx|css|json)$)': '@nrwl/react/plugins/jest',
  '^.+\\.[tj]sx?$': [
    'babel-jest',
    {
      presets: [
        [
          '@nrwl/next/babel',
          {
            'preset-react': {
              runtime: 'automatic',
              importSource: '@emotion/react',
            },
          },
        ],
      ],
      plugins: ['@emotion/babel-plugin'],
    },
  ],
},
```